config/locales/fr.yml

```diff
@@ -13,26 +34,58 @@ fr:
     people_followed_by: Personnes suivies par %{name}
     people_who_follow: Personnes qui suivent %{name}
     posts: Statuts
+    remote_follow: Suivre à distance
     unfollow: Ne plus suivre
   application_mailer:
+    settings: 'Changer les préférences e-mail: ${link}'
     signature: Notifications de Mastodon depuis %{instance}
+    view: 'Voir:'
+  applications:
+    invalid_url: L'URL fournie est invalide
   auth:
     change_password: Changer de mot de passe
     didnt_get_confirmation: Vous n’avez pas reçu les consignes de confirmation ?
     forgot_password: Mode passe oublié ?
     login: Se connecter
+    logout: Se déconnecter
     register: S’inscrire
     resend_confirmation: Envoyer à nouveau les consignes de confirmation
     reset_password: Réinitialiser le mot de passe
-    set_new_password: Établir le nouveau mot de passe
```

Comment:
I'm not sure, "Etablir" doesn't sound very usual in French, I never saw it in French UIs anyway.